pages/tools/fileconverter.vue

```diff
@@ -219,11 +205,11 @@
 
 <script>
 import fileDownload from 'js-file-download'
-import CloudLoading from '../../components/CloudLoading.vue'
-import geode_objects from './geode_objects'
+import CloudLoading from '@/components/CloudLoading.vue'
+import geode_objects from '../../assets/geode_objects'
```

Comment:
```suggestion
import geode_objects from '@/assets/geode_objects'
```
Use relative path only when files are physically related, like a component and its sub-component.